package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "google-map-react",
-  "version": "1.0.5",
+  "name": "@jfw/google-map-react",
```

Comment:
Lets remove the @jfw

Reply:
Sorry for that. I needed to publish the component in order to use it in a project. I'll remove it on `master`.